[2024-01-15T09:00:05.597+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: data_ingestion.load_data scheduled__2024-01-14T09:00:00+00:00 [queued]>
[2024-01-15T09:00:05.601+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: data_ingestion.load_data scheduled__2024-01-14T09:00:00+00:00 [queued]>
[2024-01-15T09:00:05.601+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 1
[2024-01-15T09:00:05.606+0000] {taskinstance.py:1382} INFO - Executing <Task(_PythonDecoratedOperator): load_data> on 2024-01-14 09:00:00+00:00
[2024-01-15T09:00:05.611+0000] {standard_task_runner.py:57} INFO - Started process 64702 to run task
[2024-01-15T09:00:05.613+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'data_ingestion', 'load_data', 'scheduled__2024-01-14T09:00:00+00:00', '--job-id', '229', '--raw', '--subdir', 'DAGS_FOLDER/fetch-api-data.py', '--cfg-path', '/tmp/tmp7_raxjiw']
[2024-01-15T09:00:05.614+0000] {standard_task_runner.py:85} INFO - Job 229: Subtask load_data
[2024-01-15T09:00:05.634+0000] {task_command.py:416} INFO - Running <TaskInstance: data_ingestion.load_data scheduled__2024-01-14T09:00:00+00:00 [running]> on host f28e5db0fcbc
[2024-01-15T09:00:05.725+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='data_ingestion' AIRFLOW_CTX_TASK_ID='load_data' AIRFLOW_CTX_EXECUTION_DATE='2024-01-14T09:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-01-14T09:00:00+00:00'
[2024-01-15T09:00:05.731+0000] {base.py:73} INFO - Using connection ID 'postgres_conn' for task execution.
[2024-01-15T09:00:05.832+0000] {logging_mixin.py:154} INFO - country_holiday already exists in the database. Skip data loading.
[2024-01-15T09:00:05.833+0000] {python.py:194} INFO - Done. Returned value was: None
[2024-01-15T09:00:05.863+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=data_ingestion, task_id=load_data, execution_date=20240114T090000, start_date=20240115T090005, end_date=20240115T090005
[2024-01-15T09:00:05.911+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2024-01-15T09:00:05.957+0000] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
